src/Html.php

```diff
@@ -6451,9 +6458,14 @@ public static function getCoreVariablesForJavascript(bool $full = false)
         }
         $plugins_path = 'var GLPI_PLUGINS_PATH = ' . json_encode($plugins_path) . ';';
 
+        $session_glpi = "var SESSION_GLPI = {
```

Comment:
I gladly take opinions on this particular exposition

Reply:
I am not sure it is a good idea to expose user ID, even if it is probably already exposed by debug bar information.
Maybe a name-based UID would permit to have a stable unique identifier for a given user, without exposing sensitive information.
```
\Ramsey\Uuid\Uuid::uuid5(\Ramsey\Uuid\Uuid::NIL, sprintf('%d-%s', $_SESSION["glpiID"], $_SESSION["glpiname"]));
```